### PR TITLE
disable login detection

### PR DESCRIPTION
**Pull request by brindy**
Task/Issue URL: https://app.asana.com/0/392891325557410/1164681539353125
Tech Design URL:
CC:

**Description**:

Emergency fix to lock down login detection.  Existing code is still mostly in place, just disabled for now.  Will tidy up/remove/fix as part of follow up project.

**Steps to test this PR**:

1. Migrate from previous version - ensure that previously detected domains and cookies are removed (won't affect other sites that are logged in but not detected)
1. Fireproof a website using the menu item - should stay logged in after using the fire button / auto-clear
1. Ensure settings still allows remove / remove all of domains

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)

### CODE REVIEW

File: DuckDuckGo/PreserveLoginsSettingsViewController.swift [hunk 1]
```diff
@@ -28,6 +28,13 @@ protocol PreserveLoginsSettingsDelegate: NSObjectProtocol {
 
 class PreserveLoginsSettingsViewController: UITableViewController {
     
+    struct Sections {
```
Comment (tomasstrba) [inline DuckDuckGo/PreserveLoginsSettingsViewController.swift]: No need to change, but isn't `enum` better for this?

**Review by tomasstrba — APPROVED** [review]
LGTM! 👍 I like "fireproof" idea

File: DuckDuckGo/PreserveLoginsSettingsViewController.swift (hunk 1 above)
Comment (brindy) [reply DuckDuckGo/PreserveLoginsSettingsViewController.swift]: Yeah, I was thinking that, but didn't want the complexity of converting sections to an enum.  I can test against the rawValue though so yeah it's a good suggestion.

File: DuckDuckGo/PreserveLoginsAlert.swift [hunk 2]
```diff
@@ -22,6 +22,17 @@ import Core
 
 class PreserveLoginsAlert {
     
+    static func showConfirmFireproofWebsite(usingController controller: UIViewController, completion: @escaping() -> Void) {
```
Comment (bwaresiak) [inline DuckDuckGo/PreserveLoginsAlert.swift]: I'd suggest naming tweak: Looking at the flow, this is not `completion` but `onConfirm` handler.

File: DuckDuckGo/PreserveLoginsSettingsViewController.swift (hunk 1 above)
Comment (brindy) [reply DuckDuckGo/PreserveLoginsSettingsViewController.swift]: pushed a change there - feel free to sanity check it. 

thanks @bwaresiak for the suggestions via chat :)

File: DuckDuckGo/PreserveLoginsSettingsViewController.swift [hunk 3]
```diff
@@ -116,19 +111,15 @@ class PreserveLoginsSettingsViewController: UITableViewController {
...
-    }
-    
-    override func tableView(_ tableView: UITableView, titleForFooterInSection section: Int) -> String? {
-        return section == 0 ? UserText.preserveLoginsSwitchFooter : nil
+        return Section(rawValue: section) == Section.domainList ? UserText.preserveLoginsDomainListHeaderTitle : nil
     }
     
     override func tableView(_ tableView: UITableView, canEditRowAt indexPath: IndexPath) -> Bool {
-        return indexPath.section == 1 && !model.isEmpty
+        return indexPath.isInSection(section: Section.domainList) && !model.isEmpty
```
Comment (bwaresiak) [inline DuckDuckGo/PreserveLoginsSettingsViewController.swift:118]: `Section` can be omitted, right?

**Review by bwaresiak — APPROVED** [review]
LGTM! Good job, two cosmetic comments inline.

File: DuckDuckGo/PreserveLoginsSettingsViewController.swift (hunk 3 above)
Comment (brindy) [reply DuckDuckGo/PreserveLoginsSettingsViewController.swift:118]: No - when in edit mode we show an extra section with the "remove all" button.

Comment (bwaresiak) [reply DuckDuckGo/PreserveLoginsSettingsViewController.swift:118]: I meant that 
`
return indexPath.isInSection(section: Section.domainList) && !model.isEmpty
`
can be changed to
`
return indexPath.isInSection(section: .domainList) && !model.isEmpty
`
😄

**Pull request merged**